run_cdc_tests.sh

```diff
@@ -20,4 +20,4 @@ set -e
 source .env/bin/activate
 export FLASK_ENV=webdriver
 
-python3 -m pytest server/webdriver/cdc_tests/
+python3 -m pytest --reruns 2 server/webdriver/cdc_tests/
```

Comment:
Curious about the 2 reruns. Is this because we have more tests now and this mitigates some of the flakiness?

Reply:
Yes it's also mainly the type of tests. There are a few tests that have some flakiness (think it was the scatter visualization ,and those for which we enter the settings manually). They're also the ones that tend to flake for base dc.
Base DC tests have a 2 reruns already enabled. So since it's the same tests and there was some flakiness, I matched the reruns.